nginx_reloader.py

```diff
@@ -63,19 +75,23 @@ def populate_hosts(fname):
         if not validate_ip(s):
           servers.append(s)
   return servers
-        
+
 def resolve_dns(servers):
   servers_dict = {}
   myResolver = dns.resolver.Resolver() #create a new instance named 'myResolver'
   for server in servers:
+    if len(include_list) > 0 and server not in include_list:
```

Comment:
This will ignore all hosts from the upstream config.  Shouldn't the include list addition to upstream config file?